cluster-scope/base/subscriptions/ocs-operator/subscription.yaml

```diff
@@ -4,9 +4,9 @@ kind: Subscription
 metadata:
   name: ocs-operator
 spec:
-  channel: stable-4.5
+  channel: stable-4.6
```

Comment:
While I know this is a WIP pr, I'm gonna comment on it anyways: you shouldn't change this in base, since base is used in the MOC CNV overlay as well. CNV is currently running 4.5, plan is to move it to 4.7 directly this week - therefore we need this to be patched in overlays instead.

Reply:
Btw, you can use https://github.com/operate-first/apps/pull/288 as a reference